app/models/course.rb

```diff
@@ -12,6 +12,19 @@ class Course < ApplicationRecord
     EHCO: "Early Headship Coaching Offer",
   }.with_indifferent_access.freeze
 
+  COURSE_ECF_ID_TO_IDENTIFIER_MAPPING = {
```

Comment:
you should already have these mappings in the database on `Course` so you're duplicating here. you'd only want to want to write this if you wanted to to save yourself the DB query

Reply:
The alternative would be shift everything in the database to in memory. i recommend one strategy and not have both in place

Reply:
I couldn't find the identifier in the NPQ app and wanted to avoid any db columns as I was planning to see if we can replace the `identifier` in the ECF API with the `ecf_id`.